Example/Bellerophon/BellerophonModel.swift

```diff
@@ -7,33 +7,46 @@
 //
 
 import Bellerophon
-import ObjectMapper
+
+typealias JSON = [String: Any]
 
 //{
 //    "apiInactive": false,
 //    "forceUpdate": false,
 //    "retryInterval": null,
 //    "userMessage": null
 //}
-class 💩: Mappable, BellerophonObservable {
-
-    var isAPIInactive: Bool = false
-    var shouldForceUpdate: Bool = false
-    var interval: TimeInterval = 0
-    var userMessageStr: String = ""
-
-    required init?(map: Map) {
-        isAPIInactive <- map["apiInactive"]
-        shouldForceUpdate <- map["forceUpdate"]
-        interval <- map["retryInterval"]
-        userMessageStr <- map["userMessage"]
-    }
+class BellerophonModel: BellerophonObservable {
+
+    let isAPIInactive: Bool
+    let shouldForceUpdate: Bool
+    let interval: TimeInterval
+    let userMessageStr: String
+
+    required init?(json: JSON) {
+        if let apiInactive = json["apiInactive"] as? Bool {
+            self.isAPIInactive = apiInactive
```

Comment:
I would prefer to use `self.isAPIInactive = json["apiInactive"] as? Bool ?? false`. But that's just an idea 😄